concordium-sdk/src/main/java/com/concordium/sdk/crypto/wallet/web3Id/Statement/RequestStatement.java

```diff
@@ -13,4 +16,22 @@ public class RequestStatement {
     private String id;
     private List<String> type;
     private List<AtomicStatement> statement;
+
+    public List<AtomicStatement> getUnsatisfiedStatements(IdentityObject identityObject) {
+        return statement.stream().filter(s -> {
+            try {
+                return !s.canBeProvedBy(identityObject);
```

Comment:
I am really not a fan of `canBeProvedBy` throwing exceptions. I think that should be contained in the internal implementation.

Reply:
I have not removed this, but I have made it more clear for this and RequestStatement's `canBeProvenBy` when it is because of incorrect statement type.